core/shared/src/main/scala/zio/Runtime.scala

```diff
@@ -174,4 +173,45 @@ object Runtime {
     val environment = r
     val platform    = platform0
   }
+
+  lazy val default = Runtime((), Platform.default)
+
+  lazy val global = Runtime((), Platform.global)
+
+  /**
+   * Builds a new runtime given an effectual and resourceful environment `R`
+   * and the default platform. This method is effectful because it executes
+   * acquiring the environment. Returns a finalizer that can be used to release
+   * the environment.
+   */
+  def unsafeDefault[R <: Has[_]](layer: ZLayer.NoDeps[Nothing, R]): (Runtime[R], Exit[Any, Any] => UIO[Any]) =
+    unsafeMake(layer, default.platform)
+
+  /**
+   * Builds a new runtime given an effectual and resourceful environment `R`
+   * and Scala's global execution context. This method is effectful because
+   * it executes acquiring the environment. Returns a finalizer that can be
+   * used to release the environment.
+   */
+  def unsafeGlobal[R <: Has[_]](layer: ZLayer.NoDeps[Nothing, R]): (Runtime[R], Exit[Any, Any] => UIO[Any]) =
+    unsafeMake(layer, global.platform)
+
+  /**
+   * Builds a new runtime given an effectual and resourceful environment `R`
+   * and a [[zio.internal.Platform]]. This method is effectful because it
+   * executes acquiring the environment. Returns a finalizer that can be used
+   * to release the environment.
+   */
+  def unsafeMake[R <: Has[_]](
```

Comment:
We usually mark unsafe methods as package-private; will this be needed outside of the library?

Reply:
Unfortunately this will be needed outside the library. The background here is that with the new environment encoding `Runtime.default` returns a `Runtime[Unit]` instead of a `Runtime[ZEnv]`, which means that if you have a mixed application and you want a runtime to unsafely run effects that use any type of environment you no longer have one "off the shelf". The goal of the PR was to make that use case easier and since all environments are managed now we need to unsafely run them to get a runtime with an environment out.

We could only offer the `unsafeDefault` and `unsafeGlobal` methods above but I don't think that is really any safer and it just forces the user to use `mapPlatform` if they want to use their own executor settings.